packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -683,19 +689,28 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
 
     private checkSnapshotUrl() {
         if (!this.snapshotUrl) {
-            throwOdspNetworkError("methodNotSupportedBecauseNoSnapshotUrlWasProvided", 400);
+            throw new NonRetryableError(
+                "noSnapshotUrlProvided",
+                "Method failed because no snapshot url was available",
+                DriverErrorType.genericError);
```

Comment:
Yes, I wanted to use DriverErrorType.usageError. But it will take 3 PRs to make it happen :) Plus waiting couple releases to make sure hosts add code on their side to handle it. So opened https://github.com/microsoft/FluidFramework/issues/8489 to make it happen in future.